### DISCUSSION TITLE
storage/hwameistor/install/post-check/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1688] [Announcements]
# storage/hwameistor/install/post-check/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/storage/hwameistor/install/post-check/

**Comment by liyuerich** [comment]
kubectl -n hwameistor get pod 检查出来的结果已经有变化，需要更新

**Comment by liyuerich** [comment]
kubectl get storageclass hwameistor-storage-disk-hdd
Error from server (NotFound): storageclasses.storage.k8s.io "hwameistor-storage-disk-hdd" not found

**Reply by liyuerich** [reply]
这个报错已经知晓原因，请忽视

**Reply by liyuerich** [reply]
今天重新安装，检查后发现  storageclass 名字是这个。hwameistor-storage-lvm-hdd
